OpenRA.Mods.Common/Traits/Multipliers/ProductionTimeMultiplier.cs

```diff
@@ -0,0 +1,44 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Modifies the production time of this actor for a specific queue or when a prerequisite is granted.")]
+	public class ProductionTimeMultiplierInfo : TraitInfo<ProductionTimeMultiplier>, IProductionTimeModifierInfo
+	{
+		[Desc("Percentage modifier to apply.")]
+		public readonly int Multiplier = 100;
+
+		[Desc("After applying the multiplier, also add this to the build time.")]
+		public readonly int ExtraBuildTime = 0;
```

Comment:
Please remove this as outlined above and simplify the interface return value to `int`.